docker-compose.yml

```diff
@@ -33,6 +33,12 @@ services:
       - POSTGRES_PASSWORD=puppetdb
       - POSTGRES_USER=puppetdb
       - POSTGRES_DB=puppetdb
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready --username=puppetdb --dbname=puppetdb"]
```

Comment:
This is not sophisticated enough... will be putting up another PR shortly to remedy.

Reply:
https://github.com/puppetlabs/pupperware/pull/115 fixes this